### DISCUSSION TITLE
Welcome to firefox-translations Discussions!

### DISCUSSION THREAD

**Discussion by andrenatal** [mozilla/firefox-translations#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cpatti97100** [comment]
thank you for working on this project, it looks beautiful!

**Comment by hmm-mm** [comment]
I think you guys should consider adding a little textbox feature also, so that users can copy paste some text if they only need a few lines to be translated and not the whole page.
To increase your  extension's downloads as a firefox extension. Considering the number of anime fans increasing, you should add japanese with more priority. That alone will surge your downloads.

Overall, the project is amazing!!

**Comment by andrenatal** [comment]
Hello. That's already available. You can highlight text on the page, right
click and select "Firefox Translations" and that should be sent to a
translations popup that's accessible in your toolbar. Let me know if that
doesn't work for you, please.

**Reply by hmm-mm** [reply]
Oh yeah, it works.
But i guess some (maybe more)users like me wont be aware of it. 
I saw a line now regarding it in the release notes, but again i don't think most people would see that.

I think a couple of screenshots like these below as an addition on the (https://addons.mozilla.org/en-US/firefox/addon/firefox-translations/) webpage would do the trick. As its a good feature, the one that people could also be looking for, to meet their personal privacy ideals and convenience in an extension/tool.
1.
![image](https://user-images.githubusercontent.com/131739165/234265003-234d34c7-8f3e-4fdd-80ec-52a2bf15c975.png)
2.
![image](https://user-images.githubusercontent.com/131739165/234263820-a1aac201-8d2a-4ca7-b8f2-a295e9c1f44c.png)



And lastly I love your concept about local translation engine, keep going!!
